docs: added note about logo size (#3956)
Kotlin/dokka: docs/topics/formats/dokka-html.md

For an example of how to do this, see our
[custom format example project](https://github.com/Kotlin/dokka/tree/%dokkaVersion%/examples/gradle/dokka-customFormat-example).

> Note that the maximum supported logo dimensions are 120 px width and 36 px height. If you provide a larger image, it will be resized.

Reviewer: ```suggestion
The maximum supported logo dimensions are 120 pixels in width and 36 pixels in height. If you use a larger image, it will be automatically resized.
```
